ui/src/components/App.js

```diff
@@ -29,70 +28,76 @@ import base from 'theme';
 // Router does not detect route changes
 // https://github.com/ReactTraining/react-router/issues/6072
 const ProvidedRoutes = () => (
-  <ModalStateContext.Consumer>
-    {(modalState) => (
-      <Component
-        initialState={{
-          version: globals.VERSION,
-        }}
-        didMount={() => {
-          if (!localStorage.getItem(globals.SEEN_WARNING_KEY)) {
-            modalState.setModalState({ component: <WarningModal modalState={modalState} /> });
-          }
-        }}
-      >
-        {({ state }) => (
-          <ExpandedUnexpandedProvider>
-            <SkipNav />
-            <Switch>
-              <Route
-                path="/"
-                exact
-                render={() => (
-                  <>
-                    <Header />
-                    {/* <SearchWrapper version={state.version} index="models" /> */}
-                  </>
-                )}
-              />
-              {process.env.REACT_APP_ENABLE_ADMIN ? (
-                <Route
-                  path="/arranger"
-                  render={({ match }) => (
-                    <>
-                      <Header />
-                      <ArrangerDashboard basename={match.url} />
-                    </>
-                  )}
-                />
-              ) : (
-                ''
-              )}
-              <Route
-                path="/admin"
-                render={({ location }) => (
-                  <>
-                    <Header subheading="Searchable Catalog CMS" />
-                    <Admin location={location} />
-                  </>
-                )}
-              />
-              <Route
-                path="/model/:modelName"
-                render={({ match }) => (
-                  <>
-                    <Header />
-                    <Model modelName={match.params.modelName} />
-                  </>
-                )}
-              />
-            </Switch>
-            <Footer />
-          </ExpandedUnexpandedProvider>
+  <ArrangerDataContext.Consumer>
+    {(context) => (
```

Comment:
Use of Consumer / RootProvider is changed in https://github.com/nci-hcmi-catalog/portal/pull/1099/files